src/Core/tests/DeviceTests/Memory/MemoryTestTypes.cs

```diff
@@ -9,7 +9,9 @@ public class MemoryTestTypes : IEnumerable<object[]>
 	{
 		public IEnumerator<object[]> GetEnumerator()
 		{
+#if !ANDROID
 			yield return new object[] { (typeof(DatePickerStub), typeof(DatePickerHandler)) };
```

Comment:
We have more exhaustive tests that validate this inside controls. This test is currently redundant, and the failure currently only happens occasionally in CI. The failure is most likely due to how the test is written and not an actual bug in the bug.